dhcpv4/bsdp/bsdp_option_default_boot_image_id.go

```diff
@@ -4,35 +4,26 @@ import (
 	"fmt"
 
 	"github.com/insomniacslk/dhcp/dhcpv4"
+	"github.com/u-root/u-root/pkg/uio"
 )
 
+// OptDefaultBootImageID contains the selected boot image ID.
```

Comment:
This should be moved back down.

Reply:
That's not how Go comments work?

The diff may make it weird to read, but this is

```go
// OptDefaultBootImageID contains the selected boot image ID.
//
// Implements the BSDP option default boot image ID, which tells the client
// which image is the default boot image if one is not selected.
type OptDefaultBootImageID ...
```